no-computer.md

```diff
@@ -5,51 +5,79 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+A DSL for describing people by appearence.  This would be useful so that when you see someone
+you could describe more accurately what they look like to your friends. This
+could also be a useful way to report suspicious people the the authorities if
+there was a specific language for describing people in an organized manner.
```

Comment:
This would be very interesting. The difficulty here would be having something that is uniform. I know that many times I describe the same person differently than my friends would. That is because we focus on different characteristics - some of my friends really focus on hair style/color, some focus on their dress, and some focus on their eyes or cheeks or height or weight. This would definitely be an interesting thing to try to do but having it adopted by people and have people find it beneficial would be hard. I know I often can't find the person other people are describing. Perhaps the best DSL has been invented already - pictures/headshots? I know that is the easiest way for me to "describe" how someone looks to other people.